Rakefile

```diff
@@ -33,5 +31,10 @@ Rake::TestTask.new(:test) do |t|
   t.warning = false
 end
 
+task :coverage_report do
+  require 'launchy'
```

Comment:
I don't see worth adding this, you just have to go to the coverage folder, or do usually the gems have this?

Reply:
Thanks for reviewing! It's just a suggestion. We can skip this PR.

Reply:
Thank you for understanding, and thank you very much for your work! And your first change ("Please add a test for your change. Only refactoring...") was correct

Reply:
ok cool. I just opened a new PR with this change @MaicolBen